lib/src/model/directus_item_creation_result.dart

```diff
@@ -1,6 +1,6 @@
 class DirectusItemCreationResult<T> {
   final bool isSuccess;
-  final T? createdItem;
```

Comment:
Maybe this could become a computed property watching trying to return the first item in the array or null if the array is empty ?

Reply:
I agree. I will do it.